docs/source/sdk.rst

```diff
@@ -42,25 +42,37 @@ Creating SDK
 Creating wallet contract
 ------------------------
 
-create
-^^^^^^
+createFutureWallet
+^^^^^^^^^^^^^^^^^^
 
-**sdk.create(ensName)**
+**sdk.createFutureWallet()**
 
-  creates new wallet contract.
+prepares wallet contract deployment.
```

Comment:
Creates a FutureWallet, which contains all information required to deploy and use Wallet in the future.